src/universal/modules/outcomeCard/containers/OutcomeCard/OutcomeCardContainer.js

```diff
@@ -156,10 +180,11 @@ class OutcomeCardContainer extends Component {
           hasDragStyles={hasDragStyles}
           isAgenda={isAgenda}
           isDragging={isDragging}
-          isEditing={isEditing}
+          isEditing={isEditing()}
```

Comment:
I'm a fan of extracting expressions into functions / bound variables when it helps make things a bit clearer (e.g. `shouldAnnounceEditing` above). That said, in this case I now realize we've already got `isEditing` on the left hand side, so I guess I got a little carried away :)